SocialFilter/sections/specific.txt

```diff
@@ -75,6 +75,7 @@ mentalfloss.com,90min.com,fansided.com,90min.de,12thmanrising.com,1428elm.com,8p
 !
+ai.meta.com##._amgj > div._9bhp:has(> div._9bho)
```

Comment:
I prefer `div[class^="_"] > div[class*=" _"]:has(> div[class^="_"] > a[href*="facebook.com/sharer/sharer.php"])`.

Reply:
without space in the second div maybe?
`div[class^="_"] > div[class*="_"]:has(> div[class^="_"] > a[href*="facebook.com/sharer/sharer.php"])`